test/db/universities.spec.ts

```diff
@@ -17,6 +17,19 @@ describe('DB Universities', () => {
     emailUniversity: '',
   }];
 
+  describe('getAllOrderByName', () => {
+    it('should get all universities and order them', async () => {
+      await seed(db, true);
+
+      const universities = await dbUniversities.getAllOrderByName();
+      universities.length.should.equal(55);
+      universities
+        .map((university) => university.name)
+        .sort()
+        .forEach((university, i) => university.should.equal(universities[i].name));
```

Comment:
:thinking: is it a good idea to test with logic here?
I would have assert that the first one is "-- Aucune pour le moment"